### DISCUSSION TITLE
Importing Schemdraw

### DISCUSSION THREAD

**Discussion by ByteEOR** [PrairieLearn/PrairieLearn#8137] [Q&A] [answered]
Hello,

Yesterday I requested the `schemdraw` element to be added to PrairieLearn's Python libraries. I've noticed that the request was completed, but I still get errors while I try to import the library in `server.py`. Would anyone be able to help me on getting this library imported, please?

**Comment by jonatanschroeder** [comment, marked as answer]
If you are running it in production servers, note that the change has not yet been deployed. Stay tuned for announcements, when the system is updated it will be announced in this channel (assuming you're using the us.prairielearn.com server).

If you see this error locally, make sure to pull the latest version of the container. See
https://prairielearn.readthedocs.io/en/latest/installing/#upgrading-your-dockers-version-of-prairielearn

**Reply by nwalters512** [reply]
This change was deployed today, so `schemdraw` should now be accessible in production environments 🎉
